website/src/types/wastewaterConfig.ts

```diff
@@ -16,3 +18,193 @@ export const wastewaterConfig = {
+        symbol: '!',
+        nucleotideMutations: [],
+        aminoAcidMutations: [
+            'E119D',
+            'I223R',
+            'I427T',
+            'E119V',
+            'S247N',
+            'P458T',
+            'I223M',
+            'S247G',
+            'R293K',
+            'I223L',
+            'S247R',
+            'Q313R',
+            'N295S',
+            'I223K',
+            'R152K',
+            'D199E',
+            'H275Y',
+            'D199N',
+            'E119A',
+            'D199Y',
+            'G147R',
+            'E119G',
+            'I223V',
+            'I436N',
+            'D199G',
+        ],
+    },
+    {
+        name: 'Peramivir resistance mutations',
+        description:
+            "This mutation is associated with reduced inhibition by Peramivir, for more details see the <a class='link' href='https://www.who.int/teams/global-influenza-programme/laboratory-network/quality-assurance/antiviral-susceptibility-influenza/neuraminidase-inhibitor'>Global Influenza Programme Report</a>.",
+        symbol: '^',
+        nucleotideMutations: [],
+        aminoAcidMutations: [
+            'P458T',
+            'S247R',
+            'H275Y',
+            'Q136K',
+            'D199N',
+            'G147R',
+            'E119D',
+            'E119G',
+            'Q136R',
+            'I223R',
+            'I223K',
+            'E119V',
+            'I436N',
+            'I223V',
+            'S247N',
+        ],
+    },
+    {
+        name: 'Zanamivir resistance mutations',
+        description:
+            "This mutation is associated with reduced inhibition by Zanamivir, for more details see the <a class='link' href='https://www.who.int/teams/global-influenza-programme/laboratory-network/quality-assurance/antiviral-susceptibility-influenza/neuraminidase-inhibitor'>Global Influenza Programme Report</a>.",
+        symbol: '+',
+        nucleotideMutations: [],
+        aminoAcidMutations: [
+            'P458T',
+            'S247R',
+            'H275Y',
+            'E119A',
+            'Q136K',
+            'D199Y',
+            'E119D',
+            'E119G',
+            'S110F',
+            'Q136R',
+            'I427T',
+            'E119V',
+            'I436N',
+            'I223K',
+            'I223R',
+            'I117R',
+        ],
+    },
+];
+
+// WHO uses reference KJ609208.1 and AB124658.1, mutations here use reference CY114383.1
+// When KJ609208.1 and AB124658.1 are aligned to CY114383.1 using nextclade we see no insertions/deletions only mutations
+// Therefore, we can use CY114383.1 as the reference for the mutations
+// See https://github.com/anna-parker/NAIMutations/tree/main for calculation details
+// TODO: Some mutations are only of interest when with other mutations (e.g. deletions) but are now marked individually
```

Comment:
Is that relevant for users? If yes, we could put it on the `swiss-wastewater/index.astro` page, because we wanted to use that for docs.

Reply:
I'm not sure if users need to know - I would say probably not. I think its good to keep the description here in case we need to make changes in future as the calculation isn't trivial - but I think it is fine to just keep it as a comment for now :-)